nixos/modules/hardware/video/nvidia.nix

```diff
@@ -26,6 +26,8 @@ let
   nvidiaPersistencedEnabled =  cfg.nvidiaPersistenced;
   nvidiaSettings = cfg.nvidiaSettings;
   busIDType = types.strMatching "([[:print:]]+[\:\@][0-9]{1,3}\:[0-9]{1,2}\:[0-9])?";
+
+  ibtSupport = cfg.open || (nvidia.ibtSupport or false);
```

Comment:
```suggestion
  ibtSupport = cfg.open || nvidia.ibtSupport;
```

Reply:
This would break if someone tried to use a driver package that didnt have the `ibtSupport` attribute. Like if someone has an older pinned version of nixpkgs.

Reply:
That's a valid point.